vizro-core/hatch.toml

```diff
@@ -118,6 +119,14 @@ extra-dependencies = [
 features = ["kedro"]
 python = "3.9"
 
+[envs.tests]
```

Comment:
Can we just do this in our normal `envs.default` instead?

Reply:
I needed to add 3 additional libraries for the tests, didn't want to add it to default env

Reply:
That's totally fine, we can definitely add more packages to the default environment. The dependencies there are already test dependencies and they install so quickly now thanks to `uv` that adding more is completely fine.